src/loaders/ObjectLoader.js

```diff
@@ -53,7 +53,24 @@ Object.assign( ObjectLoader.prototype, {
 		var loader = new XHRLoader( scope.manager );
 		loader.load( url, function ( text ) {
 
-			scope.parse( JSON.parse( text ), onLoad );
+			var json = null;
+			
+			try {
+
+				json = JSON.parse( text );
+
+                        } catch( error ) {
+
+				console.log(err.message);
+				onError(loader);
```

Comment:
My bad, I mistook XHRLoader for XMLHttpRequest. At that point the XMLHttpRequest succeeded and is not accessible anymore. Should we call onError without parameters ?